gen/testdata/postgres.yml

```diff
@@ -32,36 +35,18 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: ariga/atlas-action@v0
+      - uses: ariga/setup-atlas@v0
+        with:
+          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
+      - uses: ariga/atlas-action/migrate/lint@master
         with:
-          dir: 'migrations'
+          dir: 'file://migrations'
           dir-name: 'name'
           dev-url: 'postgres://postgres:pass@localhost:5432/dev?search_path=public&sslmode=disable'
-          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
-  sync:
-    needs: lint
-    services:
-      # Spin up a postgres:15 container to be used as the dev-database for analysis.
-      postgres:
-        image: postgres:15
-        env:
-          POSTGRES_DB: dev
-          POSTGRES_PASSWORD: pass
-        ports:
-          - 5432:5432
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-start-period 10s
-          --health-timeout 5s
-          --health-retries 5
-    if: github.ref == 'refs/heads/master'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ariga/atlas-sync-action@v0
+      - uses: ariga/atlas-action/migrate/push@master
+        if: github.ref == 'refs/heads/master'
         with:
-          dir: 'migrations'
-          name: 'name'
-          dev-url: 'postgres://postgres:pass@postgres:5432/dev?search_path=public&sslmode=disable'
-          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
\ No newline at end of file
+            dir: 'file://migrations'
+            dir-name: 'name'
+            dev-url: 'postgres://postgres:pass@localhost:5432/dev?search_path=public&sslmode=disable'
```

Comment:
why did the host for the driver change to localhost?
Do you remember why it was different for lint and sync?

Reply:
There was added complexity added to support passing two parameters to the template, we can remove that extra complexity if we can always use `localhost`

Reply:
The reason it was not localhost is that we ran the previous action in docker. Now we run on "bare metal" so localhost is good.

Reply:
gotcha, so let's remove all the "args" thing that I added to support passing two parameters?